### DISCUSSION TITLE
Sharing data with other districts, areas

### DISCUSSION THREAD

**Discussion by jackieoh1** [code4recovery/12-step-meeting-list#934] [Q&A] [answered]
Hi all - 

I know there's a way to do this; however, I am having trouble finding the answer here.  Can someone direct me please?  Thank you!
Jodie S., TC, MI

**Comment by kiyote33** [comment, marked as answer]
Hi Jodie @jackieoh1, You can view a couple of videos on our [website ](https://code4recovery.org/)"How To" pages. Two that might be of interest to you are: 

1. [Use the 12 Step Meeting List on your Website](https://code4recovery.org/how-to-use-the-12-step-meeting-list), and
2. [List Meetings from Multiple Autonomous Organizations](https://code4recovery.org/how-to-list-meetings-from-multiple-organizations)
Let us know if you need more help...

**Reply by jackieoh1** [reply]
That's it! Thank you very much.  Feel free to delete this if you wish.
